campaigns/tdc/core.json

```diff
@@ -1048,7 +1048,7 @@
       "id": "$relentless_rule",
       "title": "Relentless",
       "type": "rule_reminder",
-      "text": "Some enemies in this campaign have the relentless keyword. During the enemy phase(after framework step 3.3), each enemy with the relentless keyword that has attacked this phase (even if that attack was canceled) readies and attacks the investigator(s) it is engaged with a second time."
+      "text": "Some enemies in this campaign have the relentless keyword. During the enemy phase (after framework step 3.3), each enemy with the relentless keyword that has attacked this phase (even if that attack was canceled) readies and attacks the investigator(s) it is engaged with a second time."
```

Comment:
For this one, you want to do a fine replace across all the po files for the sentence so translations maintain.

Reply:
I went ahead and update the files throughout the project, I usually just do a global find replace with vscode